strr-web/pages/registry-dashboard.vue

```diff
@@ -207,35 +219,51 @@ const updateTableRows = async () => {
   const applications = await getPaginatedApplications(paginationObject)
   if (applications) {
     totalResults.value = applications.total
-    tableRows.value = registrationsToTableRows(applications)
+    tableRows.value = await registrationsToTableRows(applications)
   }
 
   updateMaxPageResults()
   loading.value = false
 }
 
-const registrationsToTableRows = (applications: PaginatedApplicationsI): Record<string, string>[] => {
+const registrationsToTableRows = async (applications: PaginatedApplicationsI): Promise<Record<string, string>[]> => {
   const rows: Record<string, string>[] = []
-  applications.applications.forEach((application: ApplicationI) => {
-    const { header, registration: { unitAddress, primaryContact } } = application
+  for (const application of applications.applications) {
+    const { header, registration: { primaryContact, registrationType } } = application
+    const contactPerson = primaryContact?.name
+    let certificateIssued = false
+    if (header.registrationStatus === RegistrationStatusE.ACTIVE && header.registrationId) {
+      certificateIssued = await isCertificateIssued(header.registrationId.toString())
```

Comment:
Here is the same, calling API for each registration to check if it has the certificate? What if there are 500 registrations?